l10n_it_fatturapa_pec/views/company_view.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_config_settings_ftpa_pec" model="ir.ui.view">
```

Comment:
Andrebbe formattato come in https://github.com/OCA/l10n-italy/pull/1372/commits/25e5e22ee1fcd6ccd8397a2d05ad1fbbf74afc41#diff-a9b430bd610e90883ea3b84e4db64281R4